python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -268,6 +274,7 @@ def run_iteration(
                         run_request._replace(
                             tags={
                                 **run_request.tags,
+                                AUTO_MATERIALIZE_TAG: "true",
```

Comment:
add this here instead so that we don't store it repeatedly on every stored run request